packages/manager/cypress/e2e/core/account/user-permissions.spec.ts

```diff
@@ -558,4 +559,246 @@ describe('User permission management', () => {
           .click();
       });
   });
+
+  it('can add full-access users with no redirect occurs', () => {
```

Comment:
```suggestion
  it('can add users with full access', () => {
```
No need to mention the redirect in the test title, similar case for the test below. Only suggesting we change it because if we ever change the redirect behavior, we'll have to change the test titles too and that'll mess up our metrics for the test (test length, flakiness, etc.)